packages/opentelemetry-shim-opentracing/README.md

```diff
@@ -48,6 +48,11 @@ See [examples/opentracing-shim](https://github.com/open-telemetry/opentelemetry-
 
 Apache 2.0 - See [LICENSE][license-url] for more information.
 
+## Limitation
+
+The OpenTracing shim does not currently include a scope manager. 
+Therefore, scope management should be handled with external tooling.
```

Comment:
```suggestion
```

I don't think there is any external tooling to handle scope-management - it just never existed in OpenTracing JS so the shim also cannot provide that functionality.

Reply:
Noted. Thank you.